_component_design/breadcrumbs.md

```diff
@@ -10,37 +10,36 @@ status: deprecated
 resource: true
 ---
 
-Use breadcrumbs on Rackspace subpages to let the user know where they are in the heirarchy, and to allow them to jump up the heirarchy if they need to. Do not use breadcrumbs on the first layer of any multilayered module – which is to say, only use breadcrumbs on the second-level of nesting and deeper within a module.
+Use breadcrumbs on Rackspace sub-pages to let the user know where they are in the page heirarchy. Do not use breadcrumbs on the first layer of any multi-layered module, only use breadcrumbs on the second-level of nesting and deeper within a module.
 
 {% include toc.html %}
 
-
 # Breadcrumbs {% include complete.html %}
 
 ## When to use
 
-There are a few quick things to keep in mind with breadcrumbs.
+When using breadcrumbs, keep the following in mind:
 
-* Keep them brief and as minimal as possible – got deep breadcrumbs? Ask yourself why and reconsider your architecture.
-* The breadcrumb name should be the same as the page it goes to – if the page name is long, follow abbreviation instructions in specs.
-* Breadcrumbs always go on the top of the page above the header and only show up on the second level of your app and deeper.
+* Keep the breadcrumbs brief and as minimal as possible.
+* The breadcrumb name should be the same as the page it navigates to. If the page name is long, follow abbreviation instructions in specs.
```

Comment:
@technabors - "instructions in specs" - may I have the link to this please? This is a good place to include inline refs.

Reply:
Let me check with Roman on that one. He wrote the content for this page. I'll get back to you.

Reply:
There should be a section for abbreviations on the Specs and States page, but i'm not sure how your content files are structures, so not sure where to direct you for that. Should be the same page as stuff on "jump the crumb".